test/box-py/args.result

```diff
@@ -152,3 +156,28 @@ arg[2] => 2
 arg[3] => 3
 arg[4] => --help
 
+-b
+Save Tarantool bytecode: tarantool -b[options] input output
+  -l        Only list bytecode.
+  -s        Strip debug info (default).
+  -g        Keep debug info.
+  -n name   Set module name (default: auto-detect from input name).
+  -t type   Set output file type (default: auto-detect from output name).
```

Comment:
It is work in the form `-b -t <...>`? Will not `getopt()` complain about an unknown option?